custom-vu/src/main/kotlin/jces1209/vu/page/dashboard/cloud/CloudDashboardPage.kt

```diff
@@ -0,0 +1,23 @@
+package jces1209.vu.page.dashboard.cloud
+
+import com.atlassian.performance.tools.jiraactions.api.WebJira
+import jces1209.vu.page.dashboard.DashboardPage
+import org.openqa.selenium.By
+
+class CloudDashboardPage(
+    private val jira: WebJira
+) : DashboardPage(
+    driver = jira.driver) {
+    override fun createDashboard() {
+        driver.findElement(
+            By.xpath("(//button[@data-test-id='directory.dashboards-v2.create-button'])")).click()
```

Comment:
also, please review other selectors on the same thing
```suggestion
            By.xpath("//[@data-test-id='directory.dashboards-v2.create-button']")).click()
```